### ISSUE TITLE
pipegrid in V2 return condition length greater than 1 for if statement

### ISSUE THREAD

**Issue by VincentBellavance** [Ecosystem-Assessments/pipedat#38]
[Pipegrid in branch V2](https://github.com/Ecosystem-Assessments/pipedat/blob/v2/R/pipegrid.R) return an error at line 26:

```
Erreur dans if (c("bbox", "numeric", "integer") %in% class(aoi)) { : 
  la condition est de longueur > 1
```
i think it should be: class(aoi) %in% c("bbox", "numeric", "integer") instead.

**Comment by david-beauchesne** [comment]
Yup that does happen. But the output of `class()` can also be of length > 1. I think it should be:

`any(class(aoi) %in% c("bbox", "numeric", "integer"))`

**Comment by VincentBellavance** [comment]
Yep you're right!
